src/autocomplete/style.js

```diff
@@ -0,0 +1,26 @@
+import styled, { css } from 'styled-components';
+
+const DarkBg = css`
```

Comment:
You _should_ be able to just define this as a string, no need to use the css function.

Reply:
what is the `css` function used for then?

Reply:
Generally you would use the css function when defining a block of css to use on a component.
Something like
```
const darkBg = css`
  background: dark;
  color: white;
  ...
`;
```

then

```
<div className={darkBg} />
```